libs/service-portal/graphql/src/lib/queries/getCompanyRegistryCompany.ts

```diff
@@ -0,0 +1,16 @@
+import { gql } from '@apollo/client'
+import { CompanyInfoFragment } from '../fragments/companyInfo'
+
+export const COMPANY_REGISTRY_INFORMATION = gql`
```

Comment:
Better. Almost there!
Would be an improvement if this query would be moved to the `libs/service-portal/information/src/screens/Company/CompanyInfo.tsx` component. Since we don't want other components/devs to reuse this query. Other components should specifically request the `companyInfo fragment` if it's needed, since otherwise we are still introducing the same problem of making field resolvers obsolete and making unnecessary API requests when it isn't needed.

Reply:
IMO keeping the GraphQL AST inside the component makes a great documentation, similar to PropTypes.
You can see what the purpose of the component is, and what it needs in order to render. Additionally it's easy to clean up the queries and saving requests since you're just comparing the component to the AST in a single file :).

Reply:
Yeah, putting it that way, that's a solid point. 💯